packages/prism-react-renderer/src/index.ts

```diff
@@ -1,12 +1,17 @@
-import Prism from "prismjs"
-import Highlight from "./components/Highlight"
-import "./prism-langs"
+import { Prism } from "./prism-langs"
 import * as themes from "./themes"
+import { createElement } from "react"
+import { Highlight as InternalHighlight } from "./components/highlight"
+import { HighlightProps, PrismLib } from "./types"
 
-const defaultProps = {
-  Prism,
-  theme: themes.vsDark,
-}
-
-export { defaultProps, Prism, themes }
-export default Highlight
+/**
+ * Prism React Renderer requires this specific instance
+ * of Prism provided to ensure the languages are correctly loaded
+ */
+const Highlight = (props: HighlightProps) =>
+  createElement(InternalHighlight, {
+    prism: Prism as PrismLib,
+    theme: themes.vsDark,
+    ...props,
+  })
```

Comment:
We need to provide the _exact_ instance of Prism to use Highlight with the bundled types properly. Formerly we exported `prism` as a part of `defaultProps` to just pass it back in. Let's cut out the middle step. Users _could_ pass in a custom prism instance still if they want.